openai/template.ts

```diff
@@ -47,5 +47,5 @@ if(row['value']) {
 
 export const promptGPT35TurboTemplate = `
   ${promptTemplate}
-  Reply in json format only but even the jasvascript function should be in json format, just that and without non-whitespace character
+  Reply in json format only but even the javascript function should be in json format, just that and without whitespace character
```

Comment:
Let's try to iterate this part in a more concise way, something like:

```
- Reply only in JSON format
- Javascript functions should be in string format, without any whitespace characters```

of course we should check if it works